stat_vars/rff_weather_variability.mcf

```diff
@@ -0,0 +1,82 @@
+Node: dcid:StandardDeviation_DailyPrecipitation
```

Comment:
Did you have any luck trying to run `dc-import lint rff_weather_variability.mcf`?

Using the import tool here: https://github.com/datacommonsorg/import#using-import-tool

Reply:
I have not tried. I will now. Still figuring out the workflow, as you can tell. Thanks for catching all these typos. Update to come